src/java/org/apache/cassandra/index/accord/RouteJournalIndex.java

```diff
@@ -414,19 +412,49 @@ else if (e.column() == AccordJournalTable.SyntheticColumn.store_id.metadata && e
             {
                 storeId = Int32Type.instance.compose(e.getIndexValue());
             }
+            else if (e.column() == AccordJournalTable.SyntheticColumn.txn_id.metadata)
```

Comment:
i could reject exclusive as the logic is 100% inclusive... this is all synthetic anyways so can only be called with w/e the route index writes and can't come from a user.